src/bridge/cockpitinternalmetrics.c

```diff
@@ -81,8 +81,6 @@ static MetricDescription metric_descriptions[] = {
   { "disk.all.read",    "bytes", "counter", FALSE, DISK_SAMPLER },
   { "disk.all.written", "bytes", "counter", FALSE, DISK_SAMPLER },
 
-  { "network.all.rx",       "bytes", "counter", FALSE, NETWORK_SAMPLER },
```

Comment:
Ah sorry, I didn't think about that case. Reworked to keep these; I added a "deprecated" comment and a unit test.